feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go

```diff
@@ -136,7 +136,6 @@ func TestPrefixSetWithOCAgentRestart(t *testing.T) {
 	gnoi.KillProcess(t, dut, gnoi.OCAGENT, true)
 
 	v4PrefixSet = ds.GetOrCreatePrefixSet(tag3IPv4)
-	v4PrefixSet.SetMode(oc.PrefixSet_Mode_IPV4)
```

Comment:
why was this removed?  It seems unrelated to the main topic of this PR?

Reply:
it was a miss in previous PR, this line is already covered in deviations in next few lines